AK/Time.cpp

```diff
@@ -12,7 +12,13 @@
 #ifdef KERNEL
 #    include <Kernel/UnixTypes.h>
 #else
-#    include <sys/time.h>
+#    if defined(AK_OS_WINDOWS)
+#        include <WinSock2.h>
+// NOTE: Do not re-order these headers, Windows.h must be included before including profileapi.h
```

Comment:
Feels like this will be a problem in more locations than this in the future, maybe this should be added to the coding style documentation?

Reply:
Probably... I hope that we can seriously limit the blast radius of this kind of behavior though. It should be very rare in general. If anything it would be an exception footnote. Let's add it to the docs once we see it in enough places.